changelog/README.md

```diff
@@ -3,6 +3,7 @@
 ## 4th October 2024
 
 * [CHM: Update prices](/channel-manager-operations/inventory.md) extended with `dependent rates`.
+* New operation [Mews API Operations: Process availability block](../mews-operations/availabilityBlockProcessor.md).
```

Comment:
* We can just write "Mews: \<operation name\>" to show it is on the Mews side.
* We should reference the operation specifically, not just the page.
* Not sure if the operation name should be singular or plural, see later comments.

```suggestion
* New operation [Mews: Process availability block](../mews-operations/availabilityBlockProcessor.md#process-availability-block).
```